fuseclient.sh

```diff
@@ -1,12 +1,12 @@
 SERVICE_NAME="Fuse client"
-PATH_TO_JAR=target/ega-fuse-1.0-SNAPSHOT.jar
-PROCESSCNT=$(ps x |grep -v grep |grep -c "ega-fuse-1.0-SNAPSHOT.jar")
-PID=$(ps aux | grep "ega-fuse-1.0-SNAPSHOT.jar" | grep -v grep | awk '{print $2}')
+PATH_TO_JAR=target/ega-fuse-0.1.0.jar
```

Comment:
You have the variable there but then you ignore it in the rest of the script, if you use that variable you won't need to change version number on so many places every time we change the version.

Reply:
Done